plugin/build.gradle

```diff
@@ -54,13 +54,28 @@ dependencies {
     compile('org.jenkins-ci.plugins:job-dsl-core:1.66') {
         exclude(module: 'groovy-all')
     }
+    compile 'org.jenkins-ci.plugins:job-dsl:1.66@jar'
```

Comment:
Is `@jar` really required here? That should be the default, no?

Reply:
It is, the default packaging is 'hpi' for this artifact.